docs/_content/overige/functioneel/besluiten_api.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Besluiten-API"
+date: '21-01-2019'
+---
+
+# BESLUITEN API
+
+FUNCTIONELE BESCHRIJVING
+
+Hieronder lichten we de gebruiksmogelijkheden van deze API toe. Voor de
+werking van deze API verwijzen wij u naar de OAS-specificatie (zie
+‘link’ hieronder).
+
+<table>
+<tbody>
+<tr class="odd">
+<td><strong>Aspect</strong></td>
+<td><strong>Beschrijving</strong></td>
+</tr>
+<tr class="even">
+<td>Doel</td>
+<td>Het kunnen aanspreken van een voorziening voor het onderhouden en raadplegen van Besluiten inclusief relaties naar informatieobjectenobjecten.</td>
```

Comment:
informatieobjectenenobjecten?

Reply:
Woeps, ik had de cursor verkeerd staan en dan is blind typen (en daarop vertrouwen) niet handig :-P
Gefixed.